DataDog/integrations-core
# [NDM] Add NDM metadata support for Cisco ACI

### What does this PR do?

https://datadoghq.atlassian.net/browse/NDMII-2651

This PR introduces methods that will submit NDM device and interface metadata to Event Platform (specifically to our `ndm` track in EP)

See document on Confluence for additional context and reasoning: 
[Cisco ACI Integration (metadata)](https://datadoghq.atlassian.net/wiki/spaces/II/pages/3647078810/Cisco+ACI+Integration+metadata?focusedCommentId=3662938300)


### Motivation

We are looking to move existing Cisco ACI integration into NDM, this requires device and interface metadata for it to be processed and appear on our product

### Additional Notes

#### How to QA
1.  Start a ddev env with env vars pointing to staging: `ddev env start cisco_aci py3.11`
2. Edit the configuration for the Cisco ACI yaml (use the Cisco sandbox for Cisco ACI):
```
instances:
  - aci_url: https://sandboxapicdc.cisco.com
    username: admin
    pwd: "!v3G@!4@Y" # double quotes are necessary here
    tls_verify: false
```
3. `ddev env reload cisco_aci py3.11`
4. Check that the device / interface metadata is being returned / appears OK in staging


### Review checklist (to be filled by reviewers)
